test/testlib.py

```diff
@@ -314,6 +314,7 @@ def enter_page(self, path, host=None, reconnect=True):
         while True:
             try:
                 self.wait_present("iframe.container-frame[name='%s'][data-loaded]" % frame)
+                self.wait_not_visible(".curtains")
```

Comment:
Are we sure that we want to wait on this when entering a page?

A test could check the way a package handles the case when dependencies aren't installed, e.g. docker. The curtain could be tested as part of the coverage...

I could live with adding a parameter to wait for no curtain to be present. Or can we make this only catch the generic curtain?